Casks/font/font-h/font-hanamin.rb

```diff
@@ -0,0 +1,18 @@
+cask "font-hanamin" do
+  version "20170904"
+  sha256 "571cd4a09ae7da0c642d640fc2442c050aa450ebb0587a95cdd097d41a9c9572"
+
+  url "https://glyphwiki.org/hanazono/hanazono-#{version}.zip"
+  name "HanaMin"
+  homepage "https://glyphwiki.org/hanazono/"
+
+  livecheck do
+    url :homepage
+    regex(%r{href=\"hanazono-(\d*)\.zip}i)
```

Comment:
```suggestion
    regex(/href=.*hanazono[._-]v?(\d+(?:\.\d+)*)\.zip/i)
```

Please apply these same changes to the others.
- Use `[._-]` instead of `-`
- Precede the capture group with `v?`
- Use the full capture group as per our standard regexes, but with `*`.

Reply:
This regex does not work for `font-migmix-2m`, because this href is wrongly labeled.
```html
<a href="https://github.com/itouhiro/mixfont-mplus-ipa/releases/download/v2023.1123/migmix-2m-20231123.zip">migmix-2m-20200307.zip</a>
```
I'll append one double quote to this one's regex.
```
/href=.*migmix-2m[._-]v?(\d+(?:\.\d+)*)\.zip"/i
```